electroncash_gui/qt/transaction_dialog.py

```diff
@@ -48,7 +50,11 @@
 
 dialogs = []  # Otherwise python randomly garbage collects the dialogs...
 
-if False:
+should_use_freetype = False
+if sys.platform in {"cygwin", "win32"}:
+    should_use_freetype = bool(get_config().get("windows_qt_use_freetype"))
```

Comment:
I don't think this function is guaranteed to return a valid config at module import time. In fact, it may return `None`. 

If it happens to work now, that's great, but that's likely a "happy accident".  It is not guaranteed to always be the case.  

It may be that this entire decision about which sigil to use should be done at runtime lazily exactly before the sigil is drawn (but based on this `get_config()` logic you have here).

Reply:
I just added some defensive code.

Reply:
Hmm. This still is not guaranteed to be correct. As long as we are here we may as well make it correct, no?  How do you feel about refactoring this further -- eg make a `staticmethod`  of `TxDialog` that returns the correct sigil, and perhaps the `TxDialog` class calls this function and saves the sigil as an attribute at runtime each time it is constructed in `__init__`.

Reply:
OK, but why isn't it correct?

Is it because at module import time we might not have the config object yet?

So it's the same logic correct if just refactored into a `staticmethod` of `TxDialog`?

Reply:
> Is it because at module import time we might not have the config object yet?

Yes.

Reply:
Refactored.